src/main/java/com/dsec/backend/DTO/UserInfoRestricedDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.dsec.backend.DTO;
+
+public record UserInfoRestricedDTO(Integer id, String email, String firstName, String lastName) {
+}
```

Comment:
Users role should also be contained with the rest of the users information. This is unnecessary.